entries/a/anz.com.au.json

```diff
@@ -1,11 +1,10 @@
 {
   "Australia and New Zealand Banking Group": {
     "domain": "anz.com.au",
-    "url": "https://www.anz.com.au/",
-    "contact": {
-      "facebook": "ANZAustralia",
-      "twitter": "ANZ_AU"
-    },
+    "tfa": [
+      "sms"
+    ],
+    "documentation": "https://www.anz.com.au/security/protect-your-virtual-valuables/two-factor-authentication/",
```

Comment:
ANZ's 2FA seems to be more like authorization than authentication.

Their [SMS OTP page](https://www.anz.com.au/security/account-protection/sms-otp/) says:
> We will send you a SMS OTP to your mobile phone number we have on our records, to confirm certain ANZ online banking activities.

Their [bank account protection page](https://www.anz.com.au/security/account-protection/) says:
> As extra protection for your ANZ online banking activities, we may send you a SMS one-time passcode or email notification to verify a range of online banking activities.

The documentation URL says:
> A one-time password sent to your mobile phone, to provide an additional layer of security. This may be requested to verify a range of online banking activities such as updating your personal account details on Internet Banking, making Pay Anyone payments or creating a PayID.

Reply:
To be fair, most of the bank ones are an authorisation. You only use the the MFA method to authenticate changes made to your account, not actually to log into it.

Reply:
@fletcherdodd if you find any entry wrongly labeled as supporting 2FA please create a new issue and we’ll fix it as soon as we can.

Reply:
Be that as it may, [`CONTRIBUTING.md`](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#authorization) says:
> For the purposes of this site, two factor authentication is defined as any service provided as a redundant layer for account authentication. Services that provide authorization redundancy are certainly appreciated, but should not be considered two factor authentication.

Several sites have been rejected because they only offered authorization, not authentication. Examples include #6867, #6434, #6265, and #5454.